### PR TITLE
jdk8 update

### PR DESCRIPTION
**Pull request by vborisoff**

### CODE REVIEW

**Comment by mohrezaei** [comment]
Please do the following:
1) fix your line endings. the diffs are still not correct.
2) squash when it's all done.

File: reladomo/src/main/java/com/gs/fw/common/mithra/connectionmanager/DelegatingCallableStatement.java [hunk 1]
```diff
@@ -1,705 +1,717 @@
...
+    @Override
+    public <T> T getObject (int parameterIndex, Class<T> type) throws SQLException
+    {
+        return getDelegate().getObject (parameterIndex,type);
+    }
+
+    @Override
+    public <T> T getObject (String parameterName, Class<T> type) throws SQLException
+    {
+        return getDelegate().getObject (parameterName, type);
```
Comment (motlin) [inline reladomo/src/main/java/com/gs/fw/common/mithra/connectionmanager/DelegatingCallableStatement.java]: Remove whitespace before the method parameter parens here.

File: reladomo/src/main/java/com/gs/fw/common/mithra/connectionmanager/DelegatingPreparedStatement.java [hunk 2]
```diff
@@ -1,363 +1,375 @@
...
+    }
+
+    @Override
+    public void closeOnCompletion () throws SQLException
+    {
+        this.getDelegate().closeOnCompletion();
+    }
+
+    @Override
+    public boolean isCloseOnCompletion () throws SQLException
```
Comment (motlin) [inline reladomo/src/main/java/com/gs/fw/common/mithra/connectionmanager/DelegatingPreparedStatement.java]: ...and here, and anywhere else

File: .travis.yml [hunk 3]
```diff
@@ -1,13 +1,28 @@
...
+
+matrix:
+    include:
+        - os: linux
+          env: CUSTOM_JDK="openjdk8"
+
+before_install:
+    - echo 'MAVEN_OPTS="$MAVEN_OPTS -Xmx1024m -XX:MaxPermSize=512m -XX:+BytecodeVerificationLocal"' >> ~/.mavenrc
+    - cat ~/.mavenrc
+    - javac -version
```
Comment (motlin) [inline .travis.yml]: Is this maven stuff necessary? I know we do something similar in the EC build, but I don't think this build uses maven.

File: build/setenv.bat [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-@REM
-@REM    http://www.apache.org/licenses/LICENSE-2.0
-@REM
-@REM  Unless required by applicable law or agreed to in writing,
-@REM  software distributed under the License is distributed on an
-@REM  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-@REM  KIND, either express or implied.  See the License for the
-@REM  specific language governing permissions and limitations
-@REM  under the License.
+set JDK_HOME=C:\apps\jdk1.8.0_111
```
Comment (motlin) [inline build/setenv.bat]: I realize this is just a version number bump, but shouldn't these specific numbers not be in version control?

File: .travis.yml (hunk 3 above)
Comment (vborisoff) [reply .travis.yml]: samples are maven projects

**Pull request closed without merging**